lib/livebook/application.ex

```diff
@@ -189,10 +189,10 @@ defmodule Livebook.Application do
     secrets =
       for {"LB_" <> name = var, value} <- System.get_env() do
         System.delete_env(var)
-        %Livebook.Secrets.Secret{name: name, value: value, origin: :startup}
+        %Livebook.Secrets.Secret{name: name, value: value, hub_id: "personal-hub", readonly: true}
```

Comment:
We use `"personal-hub"` a lot, perhaps let's encapsulate in `Livebook.Hubs.Personal.id()`?